### ISSUE TITLE
Any Real world production applications using cz freeze ?? 

### ISSUE THREAD

**Issue by mai1x9** [marcelotduarte/cx_Freeze#1784]
I want to know if cz freeze is used by a company/ organization and the application is deployed into production where users have installed the app and have run the app. ?? In other words, companies using cz freeze to build their apps and some of the app examples that uses cz freeze and quite popular??

Is cz freeze stable and good enough to ship the app build to end users for installation in their systems??

**Comment by Fox214** [comment]
I can definitively tell you yes, its used in some semiconductor applications at least.

**Comment by mai1x9** [comment]
would be great if Any desktop based applications for windows build using python and cx freeze.

@Fox214

**Comment by Berserker66** [comment]
Yes, have been for over 5 years. Both an application that is used in German Healthcare and a FOSS application with thousands of users each.
We do run into the occasional bug, report it and usually it's fixed pretty quickly. Always test your releases, no level of stability can take that responsibility from you.

**Comment by mai1x9** [comment]
> Yes, have been for over 5 years. Both an application that is used in German Healthcare and a FOSS application with thousands of users each. We do run into the occasional bug, report it and usually it's fixed pretty quickly. Always test your releases, no level of stability can take that responsibility from you.

Thanks @Berserker66. That's sounds great. I have one question though. I have been looking around Dropbox looks like they have used py2exe. I wonder if there's any major differences between cx freeze and py2exe. Is cx freeze as stable as py2exe ?

**Comment by manojkumarragupathi** [comment]
Not sure about desktop apps. I use cx_Freeze for binarification of python fastapi services & programs for the containers running in edge devices. It is very stable.

**Comment by igortg** [comment]
We've been using cx_Freeze for desktop apps (basically Qt+Vtk) for 15yrs now, Linux and Windows (CAE tool with hundreds of customers).

**Comment by joschkazj** [comment]
Hi @igortg,

Which settings are you using to include both `vtkmodules` and `vtk.libs` in the correct location for a frozen executable?
I'm trying to package an application with VTK (via [vedo](https://vedo.embl.es/)) and Qt5 (via PySide2) and fail to solve the `ImportError` messages.
I'm using options `--include-msvcr` and `--packages vtk` (which captures `vtkmodules` but not `vtk.libs`).
I searched online but didn't find a sample file/hook for VTK so far.
Any help would be greatly appreciated!
Thanks in advance

EDIT: this was with vtk v9.3, **downgrading to v9.2.6 and including `--packages vedo,vtk,vtkmodules`** did the trick, now the frozen executable starts without `ImportError`

**Comment by KennethNielsen** [comment]
I also use cx_Freeze profesionally for a desktop application for a research instrument. I have good experiences with the stability of the final binary, but as previously mentioned, the final binary should be tested when bumping versions at least on Python and cx_Freeze it self.
